Core/Core/Transports/SQLite.cs

```diff
@@ -164,7 +164,7 @@ public void BeginWrite()
       ret[objectId] = rowFound;
     }
 
-    return ret;
+    return Task.FromResult(ret);
```

Comment:
Is there any difference between these 2? Is this just because we're not calling `await` anywhere?

Reply:
Yea, I'm not 100% on the specific runtime differences between the two. However, there is an analyser for async methods that lack an `await` 
`CS1998: This async method lacks 'await'` (tho it appears to lack documentation)

But there's some sense here,  the `async` keyword simply means "I may be using `await`"